src/transformers/pipelines.py

```diff
@@ -746,6 +746,7 @@ def __init__(
         binary_output: bool = False,
         ignore_labels=["O"],
         task: str = "",
+        group: bool = False,
```

Comment:
Can you rename this param to something like `group_entities: bool` ? I've the feeling group is a little bit too wide here :)

Reply:
Makes sense! Only challenge is there's already a function I defined as `group_entities`. 

How about I change the param to `grouped_entities: bool` instead? 😄

Reply:
I used `grouped_entities` in the latest commit. Please let me know if this ok.